src/components/search-form/SearchForm.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+
+import { getTracks } from "../../utils/tracksStorage";
+
+const SearchForm = ({ setRawText, rawText, disabled }) => {
+  const handleTracksChange = (event) => {
+    const text = event.target.value;
+    setRawText(text);
+  };
+
+  useEffect(() => {
+    if (disabled) return;
+
+    const text = getTracks();
+    setRawText(text);
```

Comment:
Work with trackStorage is spreaded between `SearchForm` and `SearchButton`. I think it is not a good practice.